terraform/lambda_iam_policy.tf

```diff
@@ -52,6 +52,7 @@ data "aws_iam_policy_document" "discord_gems_policy" {
       "dynamodb:Scan",
       "dynamodb:Query",
       "dynamodb:PutItem",
+      "dynamodb:DeleteItem",
```

Comment:
can we restrict this delete action with conditions?

Reply:
Are you suggesting to use feature toggles for this, or is there another approach you're considering?

Reply:
As in using conditions to prevent deletion of any other kind of rows other than the opt in'out related ones @shadmansakib